.github/scripts/check-engines-cli.sh

```diff
@@ -5,17 +5,25 @@ echo "-------------- Checking CLI/Engines QE Engine --------------"
 DIR=$1
 PROJECT=$2
 
-DEFAULT_CLI_QUERY_ENGINE_TYPE='library'
+DEFAULT_QUERY_ENGINE_TYPE='library'
 
 # Check to see if the env var "PRISMA_CLI_QUERY_ENGINE_TYPE" is set if not then using the default
```

Comment:
```suggestion
# Check to see if the env var "DEFAULT_QUERY_ENGINE_TYPE" is set if not then using the default
```

Reply:
But the code checks for `PRISMA_CLI_QUERY_ENGINE_TYPE`.